rust/src/fvm/externs.rs

```diff
@@ -113,6 +102,22 @@ impl Rand2 for CgoExterns {
     }
 }
 
+fn draw_randomness(
```

Comment:
Can we have one or two "fixture" tests? I.e.:

1. In lotus, explicitly generate some randomness (e.g., with a unit test).
2. Hard-code those in a test in _this_ file, and make sure this function generates the same outputs.